Library/Formula/json-glib.rb

```diff
@@ -21,6 +19,18 @@ def install
                           "--disable-dependency-tracking",
                           "--prefix=#{prefix}",
                           "--enable-introspection=yes"
-    system "make install"
+    system "make", "install"
+  end
+  test do
+    (testpath/"test.c").write <<-EOS.undent
+      #include <json-glib/json-glib.h>
+
+      int main(int argc, char *argv[]) {
+        JsonParser *parser = json_parser_new();
+        return 0;
+      }
+    EOS
+    system ENV.cc, "-I#{HOMEBREW_PREFIX}/include/json-glib-1.0", "-I#{HOMEBREW_PREFIX}/include/glib-2.0", "-I#{HOMEBREW_PREFIX}/lib/glib-2.0/include", "-I#{HOMEBREW_PREFIX}/opt/gettext/include", "test.c", "-L#{HOMEBREW_PREFIX}/lib", "-L#{HOMEBREW_PREFIX}/lib", "-L#{HOMEBREW_PREFIX}/opt/gettext/lib", "-ljson-glib-1.0", "-lgio-2.0", "-lgobject-2.0", "-lglib-2.0", "-lintl", "-o", "test"
```

Comment:
```ruby
glib = Formula["glib"]
system ENV.cc, "test.c", "-I#{include}", "-I#{glib.opt_include}", "-L#{lib}", "-L#{glib.opt_lib}", "-l...", "-o", "test"
```

You don't need add `"-L#{HOMEBREW_PREFIX}/lib", "-L#{HOMEBREW_PREFIX}/opt/gettext/lib"`.